run_markdownlint.sh

```diff
@@ -10,4 +10,4 @@ PROJECT_ROOT="$SCRIPT_DIR"
 cd "$PROJECT_ROOT" || { echo "Could not change directory"; exit 1; }
 
 # Run markdownlint-cli2 (in a Docker container for convenience)
-docker run -v $PWD:/workdir davidanson/markdownlint-cli2:v0.12.1 --config .markdownlint-cli2.yaml "#node_modules"
\ No newline at end of file
+./src/webapp/node_modules/.bin/markdownlint-cli2 --config .markdownlint-cli2.yaml "#node_modules"
```

Comment:
Maybe add a check if `node_modules` exists and exit gracefully with a warning? (The case if webapp building was opt-out and Node is not installed)